README.rst

```diff
@@ -6,62 +6,81 @@ Astropy
-used as a quick reference when you are ready to start writing or validating
-code for submission.
+To install Astropy, use:
 
-Getting started with GitHub Codespaces
---------------------------------------
+.. code-block:: bash
+
+    pip install astropy
+
+Contributing
+============
+
+The Astropy Project is made both by and for its users, so we welcome and encourage contributions of many kinds. Our goal is to keep this a positive, inclusive, successful, and growing community by abiding with the `Astropy Community Code of Conduct <https://www.astropy.org/about.html#codeofconduct>`_.
+
+More detailed information on contributing to the project or submitting feedback can be found on the `contributions page <https://www.astropy.org/contribute.html>`_. A `summary of contribution guidelines <CONTRIBUTING.md>`_ can also be used as a quick reference when you are ready to start writing or validating code for submission.
+
+Getting Started with GitHub Codespaces
+======================================
 
 Codespaces is a cloud development environment supported by GitHub. None of the Astropy build machinery depends on it, but it is a convenient way to quickly get started doing development on Astropy.
 
-To get started, create a codespace for this repository by clicking this 👇
+To get started, create a codespace for this repository by clicking this:
 
 |Codespaces|
 
-A codespace will open in a web-based version of Visual Studio Code. The `dev container <.devcontainer/devcontainer.json>`_ is fully configured with software needed for this project. Feel free to take a look at `GitHub Codespaces Support <https://support.github.com/features/codespaces>`_ page for help.
+A codespace will open in a web-based version of Visual Studio Code. The `dev container <.devcontainer/devcontainer.json>`_ is fully configured with software needed for this project. For help, see the `GitHub Codespaces Support page <https://docs.github.com/en/codespaces>`_.
 
 **Note**: Dev containers is an open spec which is supported by `GitHub Codespaces <https://github.com/codespaces>`_ and `other tools <https://containers.dev/supporting>`_.
 
 Supporting the Project
-----------------------
+======================
 
 |NumFOCUS| |Donate|
 
-The Astropy Project is sponsored by NumFOCUS, a 501(c)(3) nonprofit in the
-United States. You can donate to the project by using the link above, and this
-donation will support our mission to promote sustainable, high-level code base
-for the astronomy community, open code development, educational materials, and
-reproducible scientific research.
+.. |NumFOCUS| image:: https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
+    :target: https://numfocus.org
+    :alt: Powered by NumFOCUS
+
+.. |Donate| image:: https://img.shields.io/badge/Donate-to%20Astropy-brightgreen.svg
+    :target: https://numfocus.salsalabs.org/donate-to-astropy/index.html
```

Comment:
Why is this repeated?

Reply:
Unintended duplication. Will be removed.